tests/table.spec.ts

```diff
@@ -31,6 +31,32 @@ async function waitForLoadingDone(page: Page) {
   ).toHaveCount(0);
 }
 
+async function dragComponent(
+  page: Page,
+  element: Locator,
+  offsetX: number,
+  offsetY: number,
+  stepNumber: number = 1000
+) {
+  // flipped the sign for offSetY since coordinates are from top-left of window
```

Comment:
I don't think you should flip the offsetY here - 0,0 being in the top left is expected for the DOM coordinate system. Bottom left is for rendering from canvas.